src/PharoThings-Devices-HD44780.package/PotHD44780Controller.class/instance/clearDisplay.st

```diff
@@ -0,0 +1,5 @@
+commands
+clearDisplay
+	self writeCommand: (instructionDic at: #LCD_CLEARDISPLAY).
+	(Delay forSeconds: 0.003) wait
+	"3 miliseconds waiting to clear the display"
```

Comment:
There is short version for waiting for delay which is exactly what you wrote in the comment:
```Smalltalk
3 milliSeconds wait
```